airbyte-cdk/bulk/toolkits/load-parquet/src/main/kotlin/io/airbyte/cdk/load/data/parquet/ParquetMapperPipelineFactory.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.data.parquet
+
+import io.airbyte.cdk.load.command.DestinationStream
+import io.airbyte.cdk.load.data.AirbyteValueNoopMapper
+import io.airbyte.cdk.load.data.MapperPipeline
+import io.airbyte.cdk.load.data.MapperPipelineFactory
+import io.airbyte.cdk.load.data.MergeUnions
+import io.airbyte.cdk.load.data.SchemalessTypesToJson
+import io.airbyte.cdk.load.data.SchemalessValuesToJson
+import io.airbyte.cdk.load.data.UnionTypeToDisjointRecord
+import io.airbyte.cdk.load.data.UnionValueToDisjointRecord
+import io.micronaut.context.annotation.Secondary
+import jakarta.inject.Singleton
+
+@Singleton
+@Secondary
```

Comment:
how does this work, beanwise? i.e. doesn't load-parquet depend on load-avro (and therefore we would have both this + the avro mapper pipeline factory on the classpath?)

Reply:
This is no longer necessary. I ended up not injecting the factories. I'll remove

Reply:
bump on removing these annotations

Reply:
I removed them in the cleanup, I didn't want to rebase again :(